setup.py

```diff
@@ -454,25 +453,30 @@ def get_exts_for(name):
             )
         )
 
-        ext_modules.append(
-            CMakeExtension("ddtrace.appsec._iast._taint_tracking._native", source_dir=IAST_DIR, optional=True)
-        )
+        ext_modules.append(CMakeExtension("ddtrace.appsec._iast._taint_tracking._native", source_dir=IAST_DIR))
 
     if platform.system() == "Linux" and is_64_bit_python():
         ext_modules.append(
             CMakeExtension(
-                "ddtrace.internal.datadog.profiling._ddup",
+                "ddtrace.internal.datadog.profiling.ddup._ddup",
                 source_dir=DDUP_DIR,
-                optional=True,
                 cmake_args=[
                     "-DPY_MAJOR_VERSION={}".format(sys.version_info.major),
                     "-DPY_MINOR_VERSION={}".format(sys.version_info.minor),
                     "-DPY_MICRO_VERSION={}".format(sys.version_info.micro),
-                    "-Ddd_wrapper_INSTALL_DIR={}".format(DDUP_DIR),
                 ],
             )
         )
 
+        # One of the stack v2 dependencies doesn't quite work on 3.7, so only support later for now
```

Comment:
Have you made a note already about which dependency is the issue?

Reply:
Will make this explicit

Reply:
Fixed in [6d56227](https://github.com/DataDog/dd-trace-py/pull/8471/commits/6d5622744a3a5e6472bb26bc15a52b97348bfa47)